Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM ubuntu:20.04 AS fuzz-introspector
```

Comment:
Could you add the proper license?

Reply:
@DavidKorczynski Added. Or it should be a different one?

Reply:
looks good but please update it to 2022

Reply:
Done